docs/modules/plugins/pages/plugin-rest-api.adoc

```diff
@@ -143,6 +143,101 @@ https://www.ietf.org/rfc/rfc3986.txt[RFC 3986].
 
 == HTTP Steps
 
+=== Set HTTP request configuration
+
+Set up custom request configuration.
+
+Available configs:
```

Comment:
put step syntax first, and move this section of step input parameter description